products/chef-workstation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Chef Workstation
+category: app
+tags: ruby-runtime
+iconSlug: chef
+permalink: /chef-workstation
+versionCommand: chef --version
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: "https://docs.chef.io/release_notes_workstation/#__LATEST__"
+eoasColumn: true
+
+identifiers:
+-   repology: chef-workstation
+
+auto:
+  methods:
+  -   chef-workstation: https://docs.chef.io/release_notes_workstation/
+
+releases:
+-   releaseCycle: "2025"
+    releaseDate: 2025-02-05
+    eoas: false
+    eol: false
+    latest: "25.5.1084"
+    latestReleaseDate: 2025-05-27
+
+-   releaseCycle: "2024"
+    releaseDate: 2024-02-20
+    eoas: false
+    eol: false
+    latest: "24.12.1073"
+    latestReleaseDate: 2024-12-18
+
+-   releaseCycle: "2023"
+    releaseDate: 2023-02-09
+    eoas: 2024-12-31
+    eol: 2024-12-31
+    latest: "23.12.1055"
+    latestReleaseDate: 2023-12-07
+
+-   releaseCycle: "2022"
+    releaseDate: 2022-01-10
+    eoas: 2024-12-31
+    eol: 2024-12-31
+    latest: "22.12.1024"
+    latestReleaseDate: 2023-01-08
+
+-   releaseCycle: "2021"
+    releaseDate: 2021-01-05
+    eoas: 2024-12-31
+    eol: 2024-12-31
+    latest: "21.12.720"
+    latestReleaseDate: 2021-12-07
+
+-   releaseCycle: "2020"
+    releaseDate: 2020-06-02
+    eoas: 2024-12-31
+    eol: 2024-12-31
+    latest: "20.8.111"
+    latestReleaseDate: 2020-08-04
+
+---
+
+> [Chef Workstation](https://docs.chef.io/workstation/) gives you everything you need to get started with Chef -
```

Comment:
We try to avoid being too "commercial" and detailed in the description, could you provide a shorter and more neutral description. See https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#text for some guidelines.

Reply:
I took a stab at a more neutral version